examples/v1beta1/trial-images/pytorch-mnist/requirements.txt

```diff
@@ -1,4 +1,4 @@
 cloudml-hypertune==0.1.0.dev6
-torch==1.13.1
-torchvision==0.14.1
+torch==2.2.1
```

Comment:
Do we need to install `torch` version in requirements if we already using `nvcr.io/nvidia/pytorch:24.01-py3` image ?

Reply:
I guess that it is no need, but I'm not sure that having separate requirements.txt would be worth it. @andreyvelich WDYT?

Reply:
Ah, I see. You meant that we should install torch and torch vision via Dockerfile, instead of requirements.txt, right?

Reply:
I thought `torch` package is already installed as part of this image `nvcr.io/nvidia/pytorch:24.01-py3`, isn't ?
I am not sure about `torchvision` tho

Reply:
Yes, both `torch` and `torchvisoin` are packed in NVIDIA image.
Let me create a separate `requirements.txt`.

Reply:
Done.

Reply:
I think you understood me. 
Since torch and torchvision is installed in those image we can remove them from `requirements.txt` and install only `cloudml-hypertune` and `Pillow`. We don't need to separate requirements for CPU and GPU images.
Also, we can remove torch installation from examples/v1beta1/trial-images/darts-cnn-cifar10/requirements.txt

Reply:
Ah, I see. Thanks.